harness/determined/common/experimental/model.py

```diff
@@ -23,6 +23,7 @@ class ModelVersion:
         model_name: (str) Name of the parent model.
```

Comment:
not your fault, but could you please fix the indentation on `model_version` while you're inside the file?